File: Light more light/light.java
import java.util.Scanner;

public class PerfectSquare {
    public static boolean isPerfectSquare(int n) {
        int root = (int) Math.sqrt(n); 
        return root * root == n;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter n: ");
        int number = sc.nextInt();

        if (isPerfectSquare(number)) {
            System.out.println("yes");
        } else {
            System.out.println("no");
        }

        sc.close();
    }
}

File: Light more light/light.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool isPerfectSquare(int n) {
    int root = (int)sqrt(n); 
    return root * root == n;
}

int main() {
    int number;
    cout << "Enter n: ";
    cin >> number;

    if (isPerfectSquare(number)) {
        cout << "yes" << endl;
    } else {
        cout << "no" << endl;
    }

    return 0;
}
